lib/java/com/google/android/material/transition/TransitionUtils.java

```diff
@@ -49,12 +54,96 @@ class TransitionUtils {
   static final int NO_DURATION = -1;
   @AttrRes static final int NO_ATTR_RES_ID = 0;
 
+  private static final int MAX_IMAGE_SIZE = 1024 * 1024;
+  
   // Constants corresponding to motionPath theme attr enum values.
   private static final int PATH_TYPE_LINEAR = 0;
   private static final int PATH_TYPE_ARC = 1;
 
   private TransitionUtils() {}
 
+  /**
+     * Creates a View using the bitmap copy of <code>view</code>. If <code>view</code> is large,
+     * the copy will use a scaled bitmap of the given view.
+     *
+     * @param sceneRoot The ViewGroup in which the view copy will be displayed.
+     * @param view The view to create a copy of.
+     * @param parent The parent of view.
+     */
+    public static View copyViewImage(ViewGroup sceneRoot, View view, View parent) {
+        Matrix matrix = new Matrix();
+        matrix.setTranslate(-parent.getScrollX(), -parent.getScrollY());
+        view.transformMatrixToGlobal(matrix);
```

Comment:
Method not available on lower APIs. Should become `ViewCompat.transformMatrixToGlobal(view, matrix);`.